Description: Change compiler defaults for raspbian
 We need armv6 and vfpv2 rather than armv7 and vfpv3-d16
Author: Peter Michael Green <plugwash@debian.org>

--- fpc-3.2.0+dfsg.orig/fpcsrc/compiler/options.pas
+++ fpc-3.2.0+dfsg/fpcsrc/compiler/options.pas
@@ -4067,25 +4067,15 @@ begin
       end;
   end;
 
+
   { ARMHF defaults }
   if (target_info.abi = abi_eabihf) then
-    { set default cpu type to ARMv7a for ARMHF unless specified otherwise }
+    { set default cpu type to ARMv6 for ARMHF unless specified otherwise }
     begin
-    {$ifdef CPUARMV6}
-      { if the compiler is built for armv6, then
-        inherit this setting, e.g. Raspian is armhf but
-        only armv6, this makes rebuilds of the compiler
-        easier }
       if not option.CPUSetExplicitly then
         init_settings.cputype:=cpu_armv6;
       if not option.OptCPUSetExplicitly then
         init_settings.optimizecputype:=cpu_armv6;
-    {$else CPUARMV6}
-      if not option.CPUSetExplicitly then
-        init_settings.cputype:=cpu_armv7a;
-      if not option.OptCPUSetExplicitly then
-        init_settings.optimizecputype:=cpu_armv7a;
-    {$endif CPUARMV6}
 
       { Set FPU type }
       if not(option.FPUSetExplicitly) then
